### PR TITLE
New package: ITK-SNAP.ITK-SNAP version 4.0.2

### PR DESCRIPTION
**Pull request by fhdrsdg**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146364)

### CODE REVIEW

**Comment by fhdrsdg** [comment]
> @fhdrsdg please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: manifests/i/ITK-SNAP/ITK-SNAP/4.0.2/ITK-SNAP.ITK-SNAP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ITK-SNAP.ITK-SNAP
+PackageVersion: 4.0.2
+InstallerType: nullsoft
```
Comment (stephengillie) [inline manifests/i/ITK-SNAP/ITK-SNAP/4.0.2/ITK-SNAP.ITK-SNAP.installer.yaml:6]:
```suggestion
InstallerType: nullsoft
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @fhdrsdg,

This package appears to have a dependency on `Microsoft.VCRedist.2015+.x64`: 

image

File: manifests/i/ITK-SNAP/ITK-SNAP/4.0.2/ITK-SNAP.ITK-SNAP.installer.yaml (hunk 1 above)
Comment (fhdrsdg) [reply manifests/i/ITK-SNAP/ITK-SNAP/4.0.2/ITK-SNAP.ITK-SNAP.installer.yaml:6]: Ah, I tried installing the manifest and running the software on my local machine and installing it in the sandbox, but I forgot to run it in the sandbox so I didn't find this dependency. Thanks for testing!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**